connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -109,6 +109,10 @@ message Join {
   Relation right = 2;
   Expression join_condition = 3;
   JoinType join_type = 4;
+  // Optional. using_columns provides a list of columns that should present on both sides of
+  // the join inputs that this Join will join on. For example A JOIN B USING col_name is
+  // equivalent to A JOIN B on A.col_name = B.col_name.
```

Comment:
let's also mention that this can't co-exist with `join_condition`

Reply:
done.